install/configuring_inventory_file.adoc

```diff
@@ -2255,7 +2255,11 @@ following variables can be used to customize the cluster console.
 |Optional base path for the cluster console. If set, it should begin and end with a slash like `/console/`. Defaults to `/` (no base path).
 
 |`openshift_console_auth_ca_file`
-|Optional CA file to use to connect to the OAuth server. Defaults to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`. Typically this does not need to be changed.
+|Optional CA file to use to connect to the OAuth server. Defaults to
+`/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`. Typically this does not
+need to be changed. You must create a ConfigMap containing the CA file and mount
+it into the `console` deployment in the `openshift-console` namespace at the
```

Comment:
You might want to use `Deployment` with a capital here since you use `ConfigMap` above